apps/condo/domains/user/components/auth/ConfirmPhoneForm.tsx

```diff
@@ -163,7 +171,7 @@ export const ValidatePhoneForm = ({ onFinish, onReset }): React.ReactElement<IVa
                         </Button>
                     </Space>
                 </Col>
-                <Col span={24}>
+                {!tokenLoading ? <Col span={24}>
```

Comment:
Please, place `<Col` on newline to match it's horizontal position with corresponding closing tag.